### PR TITLE
Remove segmentCount from the default configurations

### PR DESCRIPTION
**Pull request by adejanovski**
Update the doc with segmentCountPerNode and add token range coalescing explanations.

### CODE REVIEW

File: docs/docs/configuration/docker_vars/index.html
Comment (ossarga) [inline docs/docs/configuration/docker_vars/index.html]: This is being changed incorrectly. I may have missed this fix when I was updating the documentation.

It should be: `REAPER_LOGGING_APPENDERS_CONSOLE_LOG_FORMAT`.

Sorry about that. Should I fix this before we merge this PR?

File: src/docs/content/docs/concepts.md
Comment (ossarga) [inline src/docs/content/docs/concepts.md]: Consider the following modification to the start of the sentence.

> As of Reaper 1.2.0 and Apache Cassandra 2.2, ...

**Review by ossarga — APPROVED** [review]

**Comment by adejanovski** [comment]
Hey Anthony feel free to make all changes that seem appropriate and merge the PR.

Thanks!

**Comment by ossarga** [comment]
No worries Alex. Changes made. Waiting for CI to go green before merging.

**Pull request merged**
